hadoop-ozone/dist/src/main/smoketest/commonlib.robot

```diff
@@ -28,7 +28,7 @@ ${OM_SERVICE_ID}     om
 *** Keywords ***
 Get test user principal
     [arguments]         ${user}
-    ${instance} =       Execute                    hostname | sed 's/scm[0-9].org/scm/'
+    ${instance} =       Execute                    hostname | sed 's/scm[0-9].org/scm/;s/scm[0-9]/scm/;s/om[0-9]/om/'
```

Comment:
This was replacing `scm[0-9].org` with `scm` and ignoring everything else. It's been modified to handle `om1,om2,om3` and `scm1,scm2,scm3` as well and remove any numbers.

Reply:
Without this change, we can't run kinit on a container belonging to an OM.